src/Microsoft.SymbolStore/KeyGenerators/PortablePDBFileKeyGenerator.cs

```diff
@@ -76,11 +77,11 @@ public override IEnumerable<SymbolStoreKey> GetKeys(KeyTypeFlags flags)
         /// <param name="path">file name and path</param>
         /// <param name="pdbId">pdb guid</param>
         /// <returns>symbol store key</returns>
-        public static SymbolStoreKey GetKey(string path, Guid pdbId)
+        public static SymbolStoreKey GetKey(string path, Guid pdbId, VsPdbChecksum checksum)
```

Comment:
You can't change this public function's signature.  Need to add another function with the checksum parameter.  Giving checksum a default value won't work either.

Reply:
Never mind about the default value comment. Setting checksum = null should work.